cmd/src/debug.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+)
+
+var debugCommands commander
+
+func init() {
+	usage := `'src debug' gathers and bundles debug data from a Sourcegraph deployment for troubleshooting.
+
+Usage:
+
+	src debug command [command options]
+
+The commands are:
+
+	kube                 generates kubectl outputs
+	compose              generates docker outputs
+	server               generates docker outputs
```

Comment:
"docker outputs" are the same here. Maybe it would be more clear to say:

- dumps context from k8s deployments
- dumps context from docker-compose deployments
- dumps context from single image deployments

Reply:
✅